aidies-project/client/src/components/CampaignBrowser.js

```diff
@@ -1,47 +1,44 @@
 import React, { useState, useEffect } from 'react';
 import { Redirect, Link } from 'react-router-dom';
-
-import {getCampaigns} from '../store/campaign';
+import '../css/campaign.css'
 import { useSelector, useDispatch } from 'react-redux';
+import {getCampaigns} from '../store/campaign';
+import CampaignItems from './CampaignItems';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
 
+const useStyles = makeStyles((theme) => ({
+    
+    root: {
+        flexGrow: 1,
+    },
+    paper: {
+        padding: theme.spacing(2),
+        textAlign: 'center',
+        color: theme.palette.text.secondary,
+    },
+}));
 const CampaignBrowser = () => {
+    const classes = useStyles();
     const campaigns = useSelector(state => state.campaign.list );
     const dispatch = useDispatch();
     useEffect(() => {
         dispatch(getCampaigns())
-    }, [dispatch]);
+    }, []);
     
+    if(!campaigns.length){
+        return null;
+    }
     return (
-        <main>
-            {campaigns.map((campaign) => {
-                return (
-                    <div className="nav-entry">
-                        <Link key={campaign.id} to={`/campaign/${campaign.id}`}>
-                            <img
-                                className="nav-campaign-image"
-                                src={campaign.image}
-                                alt={campaign.name}
-                                width="300"
-                            />
-                        </Link>
-                            <div>
-                                <div className="primary-text">{campaign.name}</div>
-                                <div className="secondary-text">{campaign.summary}</div>
-                                <div className="campaign-owner">by {campaign.User.firstName} {campaign.User.lastName} for {campaign.Charity.name}</div>
-                                <div className="bid time-left">
-                                    <div className="current-bid"></div>
-                                    <div className="remaingin"></div>
-                                </div>
-                                <div className="bread-crumps">
-                                    <div className="category">{campaign.Category.name}</div>
-                                    <div className="location">{campaign.User.location}</div>
-                                    
-                                </div>
-                            </div> 
-                        </div>
-                );
-            })}
-        </main>
+        <div className={classes.root} className='campaign-browser'>
```

Comment:
Just a small style thing! I learned about a convention where people use single quotes for their JavaScript code, but double quotes for their JSX code, so I started following that convention as well!
```suggestion
        <div className={classes.root} className="campaign-browser">
```